client/src/components/OpportunityView.js

```diff
@@ -0,0 +1,184 @@
+import React, { Component } from "react";
+import { getOpportunityById, getSkillsList } from "../api/opportunities";
+import {
+  Header,
+  Grid,
+  Segment,
+  Icon,
+  Item,
+  Dimmer,
+  Dropdown,
+  Loader,
+  Message,
+  Image
+} from "semantic-ui-react";
+import moment from "moment";
+import { Link } from "react-router-dom";
+export default class OpportunityView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      opportunityId: null,
+      opportunity: [],
+      skills: [],
+      isLoading: true,
+      isEditing: false
+    };
+  }
+  //Get data and Id
+  componentWillMount() {
+    const { pathname } = window.location;
+    this.setState({
+      opportunityId: pathname && pathname.replace("/opportunities/", "")
+    });
+  }
+  getOpportunity = () => {
+    getOpportunityById(this.state.opportunityId).then(data =>
+      this.setState({ opportunity: data[0] })
+    );
+  };
+  getSkills = () => {
+    getSkillsList(this.state.opportunityId).then(data => {
+      const skillsArray = data.map(skill => skill.skill);
+      this.setState({
+        opportunity: {
+          ...this.state.opportunity,
+          skills: skillsArray
+        },
+        isLoading: false
+      });
+    });
+  };
+  componentDidMount() {
+    this.getOpportunity();
+    this.getSkills();
+  }
+
+  //Handlers
+  handleEdit = () => {
+    this.setState({ isEditing: true });
+  };
+  handleIDeleteProfile = () => {
```

Comment:
This function is not handle anything so remove it you are not using it on this PR

Reply:
i just created it to check the functionality of the button

Reply:
Yeah, that is fine but when create a PR remove.

Reply:
done